bin/theme-upload.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+const brandId = process.env.BRAND_ID;
+const { execSync } = require("child_process");
+
+function zcli(command) {
+  const data = execSync(`yarn --silent zcli ${command} --json`);
```

Comment:
Do you know if `--silent` will silence errors in case the command fails?

Reply:
We will still get the error output if the script fails. This is necessary to suppress the log during the run it doesn't interfere with the `--json` flag.